src/components/TableWithInlineEditing.js

```diff
@@ -0,0 +1,211 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { TableCell, TableRow, Table, TableHead, TableBody } from '@material-ui/core';
```

Comment:
best to change these to default imports, i.e.

import TableCell from '@material-ui/core/TableCell' etc to stop the whole library being pulled in